components/brave_new_tab_ui/containers/newTab/settings.tsx

```diff
@@ -120,7 +120,9 @@ export default class Settings extends React.PureComponent<Props, {}> {
               <SettingsText>{getLocale('brandedWallpaperOptIn')}</SettingsText>
               <Toggle
                 onChange={toggleBrandedWallpaperOptIn}
-                checked={brandedWallpaperOptIn}
+                // This option can only be enabled if
+                // users opt in for background images
+                checked={showBackgroundImage && brandedWallpaperOptIn}
```

Comment:
@karenkliu , I'm curious if user turns off the bg option, sponsored images option should have same option value.
IMO, just disabling it seems fine because its value is not changed by turning bg option off. WDYT?

Reply:
@simonhong The way we have it set up right now, turning off the Background Images toggle also turns off Sponsored Images and makes the toggle disabled until Background Images is turned back on. Settings should match that behavior.

In the future, we would like to give users more background options, so the Background Images toggle could have more sub-toggles that are independent of each other like this:

Background Images on/off
    - Brave default images on/off
    - Sponsored images on/off
    - Set your own custom images on/off
Turning off background images would turn off all the sub-toggles, but turning off Brave default images could be independent of sponsored images and not affect that toggle.

Reply:
> Turning off background images would turn off all the sub-toggles, but turning off Brave default images could be independent of sponsored images and not affect that toggle

Ok - this PR meets above requirements. Thanks for explanation!